packages/flutter/lib/src/material/search.dart

```diff
@@ -211,6 +211,16 @@ abstract class SearchDelegate<T> {
   ///
   PreferredSizeWidget? buildBottom(BuildContext context) => null;
 
+  /// Widget to display in the [AppBar] flexible space.
```

Comment:
Maybe just:

```suggestion
  /// Widget to display in the [AppBar.flexibleSpace].
```

and then remove the "see also" section?

Reply:
I did this way to keep the pattern of others documentation for build methods in the class.
You can see it in `PreferredSizeWidget? buildBottom(BuildContext)` and `List<Widget>? buildActions(BuildContext)`.

changes:
```dart
  /// Widget to display a flexible space in the [AppBar].
  ///
  /// Returns null by default, i.e. a flexible space widget is not included.
  ///
  /// See also:
  ///
  ///  * [AppBar.flexibleSpace], the intended use for the return value of this method.
  Widget? buildFlexibleSpace(BuildContext context) => null;
```